canonical-websites/www.ubuntu.com
# updated /desktop/features to vbt

## Done

* updated /desktop/features to vbt
* I can’t get the gaming section animation right… someone needs to help, it is flashing at the top of the page
* removed the open_source_phone include and embedded it
* put email on a white strip and got rid of the animated gif

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop/features)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-features.png) and [demo](http://www.ubuntu.com-1659-desktop-features.demo.haus/desktop/features)

## Issue / Card

Fixes #1659
